wp-content/plugins/inn-misc-functionality/inc/shortcode_innovation_mailchimp.php

```diff
@@ -0,0 +1,107 @@
+		</div>
+		<div class="mc-field-group">
+			<label for="mce-FNAME">First Name </label>
+			<input type="text" value="" name="FNAME" class="" id="mce-FNAME">
+		</div>
+		<div class="mc-field-group">
+			<label for="mce-LNAME">Last Name </label>
+			<input type="text" value="" name="LNAME" class="" id="mce-LNAME">
+		</div>
+		<div class="mc-field-group">
+			<label for="mce-MMERGE3">Organization </label>
+			<input type="text" value="" name="MMERGE3" class="" id="mce-MMERGE3">
+		</div>
+		<div class="mc-field-group">
+			<label for="mce-MMERGE4">Job Title </label>
+			<input type="text" value="" name="MMERGE4" class="" id="mce-MMERGE4">
+		</div>
+		<div class="mc-field-group input-group">
+			<label for="MMERGE5">Do you work for an INN member organization? </label>
+			<ul>
+				<li><input type="radio" value="Yes" name="MMERGE5" id="mce-MMERGE5-0"><label for="mce-MMERGE5-0">Yes</label></li>
+				<li><input type="radio" value="No" name="MMERGE5" id="mce-MMERGE5-1"><label for="mce-MMERGE5-1">No</label></li>
+			</ul>
+		</div>
+		<div class="mc-field-group">
+			<label for="mce-MMERGE8">What newsletter information are you interested in? </label>
+			<input type="text" value="" name="MMERGE8" class="" id="mce-MMERGE8">
+		</div>
+		<div class="mc-field-group input-group">
+			<label for="group[4101]">Who are you? </label>
+			<ul><li><input type="radio" value="1" name="group[4101]" id="mce-group[4101]-4101-0"> <label for="mce-group[4101]-4101-0">INN Member</label></li>
+				<li><input type="radio" value="2" name="group[4101]" id="mce-group[4101]-4101-1"> <label for="mce-group[4101]-4101-1">Funder</label></li>
+				<li><input type="radio" value="4" name="group[4101]" id="mce-group[4101]-4101-2"> <label for="mce-group[4101]-4101-2">I am not an INN Member/Funder, but I am interested in nonprofit news.</label></li>
+			</ul>
+		</div>
+		<div id="mce-responses" class="clear">
+			<div class="response" id="mce-error-response" style="display:none"></div>
+			<div class="response" id="mce-success-response" style="display:none"></div>
+		</div>
+
+		<div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_81670c9d1b5fbeba1c29f2865_19bec3393e" tabindex="-1" value=""></div>
+
+		<div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button"></div>
+		</div>
+	</form>
+</div>
+<script type='text/javascript' src='//s3.amazonaws.com/downloads.mailchimp.com/js/mc-validate.js'></script>
+<script type='text/javascript'>(function($) {window.fnames = new Array(); window.ftypes = new Array();fnames[0]='EMAIL';ftypes[0]='email';fnames[1]='FNAME';ftypes[1]='text';fnames[2]='LNAME';ftypes[2]='text';fnames[3]='MMERGE3';ftypes[3]='text';fnames[4]='MMERGE4';ftypes[4]='text';fnames[5]='MMERGE5';ftypes[5]='radio';fnames[6]='INDEXUSER';ftypes[6]='text';fnames[7]='INDEXPASS';ftypes[7]='text';fnames[8]='MMERGE8';ftypes[8]='text';}(jQuery));var $mcj = jQuery.noConflict(true);</script>
```

Comment:
@benlk Is it worth having this minified onto one line instead of expanding it for readability?

Reply:
I don't think lessening the file size by a newline is worth making it harder for us to read this. :/

Reply:
> what i meant to ask was would it be worth expanding that line so it's easier to read
Good point; I'll expand it.